### ISSUE TITLE
Advising the hyperparameters for training GIN for Mutagenicity

### ISSUE THREAD

**Issue by ZhaoningYu1996** [Wuyxin/ReFine#2]
Hi,

Could you provide me with the hyperparameters for training GIN for Mutagenicity?
I can not reproduce a 100% test accuracy model.

Thank you,
Zhaoning

**Comment by Wuyxin** [comment]
Hi Zhaoning!

The models reached 82~85% accuracy in this dataset (you can consider the models in the OGB repo for higher accuracy). And the hyperparameters are those set as default in ReFine/gnns/mutag_gnn.py. thx

**Comment by ZhaoningYu1996** [comment]
Thank you so much for your reply.
Your paper claims that The well-trained Graph Isomorphism Network (GIN) [26, 42] has achieved a 100% testing accuracy.
Are you using the well-trained GIN for explanation or using the 82-85% test accuracy GIN?

**Comment by Wuyxin** [comment]
The 82-85% test accuracy GIN. Sry I think this is a misclaim in the paper, we will try to clearify it, thx a lot!

But I believe this doesn't matter a lot, cuz what we do when explaining is actually using the data with right predictions. 

You can expect the complete code including the details in two weeks.

**Comment by ZhaoningYu1996** [comment]
Got it.
Thank you for your clarification.
